packages/playground/src/views/minting_view.vue

```diff
@@ -104,9 +104,15 @@
             </v-col>
             <v-col cols="10">
               <v-list-item>
-                {{ item.Minting.reward.tft.toFixed(7) }} TFT ({{ item.Minting.reward.musd.toFixed(3) }}$ at
-                {{ item.Minting.tft_connection_price.toFixed(3) }}$/TFT)</v-list-item
-              >
+                <input-tooltip
+                  inline
+                  :alignCenter="true"
+                  :tooltip="`${(item.Minting.tft_connection_price / 1e3).toFixed(3)}$/TFT`"
```

Comment:
Can we make it TFT/X$ ?

other than that it's good to merge

Reply:
As the the price of 1 TFT is 0.08 dollar in this case, it would make no sense to show it as 0.08 tft/$ because this implies that 0.08 tft is 1 dollar, however I can phrase it as follows: 
1 TFT = 0.08$
What do you think?

Reply:
Please do 🙏